scripts/install-tools.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash 
+# Copyright © 2025 Apple Inc. and the container-builder-shim project authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+GOLANGCI_LINTER_VERSION=v2.1.6
+
+install_golangci() {
+  ci_install=false
+  if [[ -x $(which golangci-lint) ]]; then
+      local version=$(golangci-lint version --short)
+      if [[ "${version}" == "${GOLANGCI_LINTER_VERSION#v}" ]]; then
+          ci_install=false
+        else
+          ci_install=true
```

Comment:
If we do something like how we install protoc, we get the check for whether to install golangci-lint for free since make will check if the file path exists, as it does for protoc [here](https://github.com/apple/container-builder-shim/blob/1941038b3b833cc6e7f030c36427a8c1cad24e8d/Protobuf.Makefile#L40). Does that make sense?